exercises/acronym/description.md

```diff
@@ -6,9 +6,12 @@ Techies love their TLA (Three Letter Acronyms)!
 
 Help generate some jargon by writing a program that converts a long name like Portable Network Graphics to its acronym (PNG).
 
+Punctuation is handled as follows: hyphens are word separators (like whitespace); all other punctuation can be removed from the input.
+
 For example:
 
 |Input|Output|
 |-|-|
 |As Soon As Possible|ASAP|
 |Liquid-crystal display|LCD|
+|Thank George It's Friday!|TGIF|
```

Comment:
The existing examples don't do it either but I think these should also be part of the tests.

Reply:
Or maybe the other way around, and I should just copy over some of the tests which demonstrate ignored punctuation?
```
|Halley's Comet|HC|
|First In, First Out|FIFO|
```

Or is it fine that the concepts are demonstrated although the exact text isn't reflected in tests?

Reply:
> Or is it fine that the concepts are demonstrated although the exact text isn't reflected in tests?

Yeah that's totally fine.